iib/workers/tasks/build.py

```diff
@@ -426,11 +427,22 @@ def _get_present_bundles(from_index, base_dir):
 
     # If no data is returned there are not bundles present
     if not bundles:
-        return []
+        return [], []
```

Comment:
To avoid re-doing the same computation later when we want just the pull_specs of the labels. I figured we can create the list when we are filtering unique bundles and then use it in the rest of the code rather than doing the same computation over and over. It'll save a few milliseconds in every build.

Reply:
thanks